integrity-backend/input-browsertrix-examples/3e11cc57daf3bad8375935cad4878123acc8d769551ff90f1b1bb0dc597-meta-content.json

```
@@ -0,0 +1,21 @@
+{
+  "contentMetadata":
+  {
+    "name": "Web archive of @rt_com Twitter",
+    "mime": "application/wacz",
+    "description": "Web archive of @rt_com Twitter on 2022-03-29",
+    "createdBy": "Starling Lab",
```

Making this closer to target registration schema based on schema.org:

```suggestion
    "author":
    {
      "@type": "Organization",
      "identifier": "https://starlinglab.org",
      "name": "Starling Lab"
    },
```